### ISSUE TITLE
Btw you have 404

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [cyberneel/blog#1]
Hi, took a look at https://cyberneel.github.io/ and noticed that the about and contact links lead to a 404. Also the page rendering seems broken.

**Comment by cyberneel** [comment]
> Hi, took a look at https://cyberneel.github.io/ and noticed that the about and contact links lead to a 404. Also the page rendering seems broken.

Yeah I need to update it, things broke because I switched to [cyberneel.com](https://cyberneel.com)
